muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/workspace/WorkspaceManagementBackingBean.java

```diff
@@ -56,9 +56,10 @@ public String init() {
       return NavigationRules.ACCESS_DENIED;
     }
     
+    workspaceChatSettings = true;
```

Comment:
Should this value be defined by what is saved in database instead of a constant?